db/snapshot/backup_task.cc

```diff
@@ -63,17 +65,30 @@ tasks::is_abortable backup_task_impl::is_abortable() const noexcept {
 
 future<tasks::task_manager::task::progress> backup_task_impl::get_progress() const {
     auto ret = _total_progress;
-    ret.completed = _progress.uploaded;
+    // _total_progress.completed before _sharded_worker is stopped
```

Comment:
A verb is missing. Like "_total_progress.completed _is updated_ before _shared_worker is stopped ..." or something. But to be honest, I don't yet understand what's happening with progress accounting in this PR